### PR TITLE
German translation for v1.0.0

### PR DESCRIPTION
**Pull request by Art4**

### CODE REVIEW

File: source/de/1.0.0/index.html.haml [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+    %li
+      Änderungen der selben Art sollten in Bereiche gruppiert werden.
+    %li
+      Versionen und Bereiche sollten verlinkt werden können.
+    %li
+      Die neuste Version kommt als erstes.
+    %li
+      Das Release-Datum jeder Version muss angegeben sein.
+    %li
+      Gib ab, ob du dich an die #{link_to "Semantische Versionierung", semver} hältst.
```
Comment (Capenus) [inline source/de/1.0.0/index.html.haml]: Gib **an**, ob

File: source/de/1.0.0/index.html.haml [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+  %p
+    Wenn der Endnutzer auf eine neue Version upgradet, sollte ihm unmissverständlich
+    klar gemacht werden, wenn etwas kaputt gehen wird. Es sollte immer möglich sein,
+    zu einer Version zu upgraden, die die zu entfernenden Features auflistet, um so
+    in seinem Source Code auf diese Features zu verzichten. Anschließend sollte man 
+    auf eine Version upgraden können, in der die Features entfernt wurden.
+
+  %p
+    Auch wenn du sonst nichts geändert hast, liste trotzdem alle veralteten und
+    entfernten Features, sowie jede funktionsgefährdende Änderung in deinem Changlog auf.
```
Comment (Capenus) [inline source/de/1.0.0/index.html.haml]: deinem **Changelog** auf.

File: source/de/1.0.0/index.html.haml [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+  %h4#confusing-dates
+    %a.anchor{ href: "#confusing-dates", aria_hidden: "true" }
+    Verwirrende Datumsformate
+
+  %p
+    In den USA setzen die Menschen den Monat an den Anfang eines Datums 
+    (<code>06-02-2012</code> für den 2. Juni 2012), wohingegen viele Menschen
+    im Rest der Welt ein roboterhaft aussehendes <code>2 June 2012</code>
+    schreiben, aber es völlig unterschiedlich aussprechen. <code>2012-06-02</code>
+    folgt der Logik vom größten zum kleinsten Wert, kann nicht mit anderen Formate
```
Comment (Capenus) [inline source/de/1.0.0/index.html.haml]: anderen **Formaten**

File: source/de/1.0.0/index.html.haml [hunk 4]
```diff
@@ -0,0 +1,320 @@
...
+    verwechselt werden und ist ein #{link_to "ISO Standard", iso}. Deshalb
+    ist es das empfohlene Datumsformat in Changelogs.
+
+  %aside
+    Es gibt sicher noch mehr. Hilf mir alle schlechten Tipps zu sammeln, indem du
+    There’s more. Help me collect these antipatterns by
```
Comment (Capenus) [inline source/de/1.0.0/index.html.haml]: This sentence seems duplicate?!

File: source/de/1.0.0/index.html.haml [hunk 5]
```diff
@@ -0,0 +1,320 @@
...
+    Prinzipiell sind #{link_to "GitHub Releases", ghr} eine gute Sache. 
+    Sie können dazu benutzt werden, um einfache Git Tags (zum Beispiel einen 
+    Tag namens <code>v1.0.0</code>) mit vielen Hinweisen zum Release 
+    auszustatten, indem man sie manuell bearbeitet, oder die Änderungen in eine
+    Git Tag Nachricht schreibt, wo sie durch GitHub automatisch in die 
+    Release-Notizen gesetzt werden.
+
+  %p
+    Leider lassen sich GitHub Releases aber nicht so einfach exportieren 
+    und sind nur über GitHub selber zu lesen. Man kann sie auch so gestaltet,
```
Comment (Capenus) [inline source/de/1.0.0/index.html.haml]: so **gestalten**,

File: source/de/1.0.0/index.html.haml [hunk 6]
```diff
@@ -0,0 +1,320 @@
...
+    Können Changelogs automatisiert ausgelesen werden?
+
+  %p
+    Es ist schwierig, weil Menschen meist unterschiedliche Formate und
+    Dateinamen verwenden.
+
+  %p
+    #{link_to "Vandamme", vandamme} ist ein Ruby gem erstellt vom
+    #{link_to "Gemnasium", gemnasium} Team, das viele (aber nicht alle)
+    Changelogs von Open Source Project einlesen kann.
```
Comment (Capenus) [inline source/de/1.0.0/index.html.haml]: **Open-Source-Projekten**

File: source/de/1.0.0/index.html.haml (hunk 4 above)
Comment (Art4) [inline source/de/1.0.0/index.html.haml]: I forgot to delete the english line, thanks.

**Comment by olivierlacan** [comment]
Thanks folks!

**Pull request merged**